src/components/AvatarGroup/helpers/abbreviateNumber.js

```diff
@@ -1,4 +1,7 @@
 export default function abbreviateNumber(number) {
+    if (!number || number < 0) {
+        throw Error('Invalid number');
```

Comment:
We usually think that it is better to not fail at all and instead show a console error and return some value like an empty string, undefined, or null. Since this function always return a string we should return an empty string here.